### PR TITLE
Rewrote documentation about approximated types

### PR DESCRIPTION
**Pull request by olsen232**
In line with rcoup's comments in last PR:

https://github.com/koordinates/sno/pull/369

### CODE REVIEW

File: docs/GPKG_WC.md [hunk 1]
```diff
@@ -51,4 +47,4 @@ According to the [GPKG specification](http://www.geopackage.org/spec/), a valid
 * `TEXT`
 * `TINYINT` (int8)
 
-There are three types that Sno supports that do not have an equivalent on this list: `interval`, `numeric`, and `time`. These types are "approximated" as `TEXT` in the GPKG working copy while keeping their original type in the Sno dataset. Sno creates a column of type `TEXT` in the appropriate place in the GPKG, and when the working copy is committed, Sno acts as if those columns have the original type (eg `interval`) instead of their actual type, `TEXT`. Since the change from (eg) `interval` to `TEXT` is just a limitation of the working copy, this apparent change in type will not show up as a change in the commit log, nor will it show up as an uncommitted change if you run `sno status` to see what local changes you have made but not yet committed.
+There are three types that Sno supports that do not have an equivalent on this list: `interval`, `numeric`, and `time`. These types are "approximated" as `TEXT` in the GPKG working copy. See APPROXIMATED_TYPES.md for more information.
```
Comment (rcoup) [inline docs/GPKG_WC.md:50]:
```suggestion
There are three types that Sno supports that do not have an equivalent on this list: `interval`, `numeric`, and `time`. These types are "approximated" as `TEXT` in the GPKG working copy. See [APPROXIMATED_TYPES](APPROXIMATED_TYPES.md) for more information. SQLite has a series of [date and time functions](https://sqlite.org/lang_datefunc.html) available which work with ISO8601 date and time strings.
```

File: docs/POSTGIS_WC.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Almost all geospatial data can be converted to PostGIS format without losing any
 
 #### Approximated types
 
-There is one type that Sno supports that has no PostGIS equivalent - an 8-bit integer. This type is "approximated" as a `SMALLINT` (which has 16 bits) in the PostGIS working copy while keeping its original type in the Sno dataset. Sno creates a column of type `SMALLINT` in the appropriate place in the GPKG, and when the working copy is committed, Sno acts as if those columns have the original type, 8-bit integer, instead of their actual type, `SMALLINT`. Since the change from 8-bit integer to `SMALLINT` is just a limitation of the working copy, this apparent change in type will not show up as a change in the commit log, nor will it show up as an uncommitted change if you run `Sno status` to see what local changes you have made but not yet committed.
+There is one type that Sno supports that has no PostGIS equivalent - an 8-bit integer. This type is "approximated" as a `SMALLINT` (which has 16 bits) in the PostGIS working copy. See APPROXIMATED_TYPES.md for more information.
```
Comment (rcoup) [inline docs/POSTGIS_WC.md:46]:
```suggestion
There is one type that Sno supports that has no PostGIS equivalent - an 8-bit integer. This type is "approximated" as a `SMALLINT` (which has 16 bits) in the PostGIS working copy. See [APPROXIMATED_TYPES](APPROXIMATED_TYPES.md] for more information.
```

File: docs/SQL_SERVER_WC.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ Almost all geospatial data can be converted to SQL Server format without losing
 
 #### Approximated types
 
-There is one type that Sno supports that has no SQL Server equivalent - the `interval`. This type is "approximated" as a string (or more precisely, as an `NVARCHAR`) in the SQL Server working copy while keeping its original type in the Sno dataset. Sno creates a column of type `NVARCHAR` in the appropriate place in the database table, and fills it with with the intervals formatted as [ISO8601 durations](https://en.wikipedia.org/wiki/ISO_8601#Durations). When the working copy is committed, Sno converts the contents of those columns back to `interval`, instead of their actual type, `NVARCHAR`. Since the change from `interval` to `NVARCHAR` is just a limitation of the working copy, this apparent change in type will not show up as a change in the commit log, nor will it show up as an uncommitted change if you run `sno status` to see what local changes you have made but not yet committed.
+There is one type that Sno supports that has no SQL Server equivalent - the `interval`. This type is "approximated" as `NVARCHAR` in the SQL Server working copy. See APPROXIMATED_TYPES.md for more information.
```
Comment (rcoup) [inline docs/SQL_SERVER_WC.md:47]:
```suggestion
There is one type that Sno supports that has no SQL Server equivalent - the `interval`. This type is approximated as `NVARCHAR` in the SQL Server working copy. See [APPROXIMATED_TYPES](APPROXIMATED_TYPES.md) for more information.
```

File: docs/SQL_SERVER_WC.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ Almost all geospatial data can be converted to SQL Server format without losing
...
 
 #### Geometry types
```
Comment (rcoup) [inline docs/SQL_SERVER_WC.md:49]: worth adding something to say the Geography type isn't currently supported but we're planning on it?

**Review by rcoup — APPROVED** [review]
This is great 👍

File: docs/GPKG_WC.md (hunk 1 above)
Comment (olsen232) [reply docs/GPKG_WC.md:50]: Done

File: docs/SQL_SERVER_WC.md (hunk 3 above)
Comment (olsen232) [reply docs/SQL_SERVER_WC.md:47]: Done

File: docs/POSTGIS_WC.md (hunk 2 above)
Comment (olsen232) [reply docs/POSTGIS_WC.md:46]: Done

File: docs/SQL_SERVER_WC.md (hunk 4 above)
Comment (olsen232) [reply docs/SQL_SERVER_WC.md:49]: Done

**Pull request merged**